de.dlr.sc.virsat.model.calculation.ui/src/de/dlr/sc/virsat/model/calculation/ui/editor/EquationSectionUriEditorInput.java

```diff
@@ -345,21 +340,7 @@ public void saveResource() {
 		if (editingDomain.isDirty(virSatResource)) {
 			// Doing the save in a workspace operation is intended, to not let

```

Comment:
Guess these comments are outdated then? Is that no concern anymore? "to not let the automatic builders jump on these files in between"

Reply:
The saveAll of the editing domain was changed to lock the workspace before doing anything else (which is why the workspace modify operation is no longer needed), so I do think the comment can also go, yes.
I will quickly remove it.